addons/point_of_sale/tools/posbox/configuration/posbox_update.sh

```diff
@@ -1,6 +1,13 @@
 #!/usr/bin/env bash
 
 sudo mount -o remount,rw /
-sudo git --work-tree=/home/pi/odoo/ --git-dir=/home/pi/odoo/.git pull
+
+# As we got an error of diverged branches in the case of git pull,
```

Comment:
The branches do not diverge actually, it is a shallow clone.

Reply:
I got an error talking about diverge.

Reply:
Because you have 2 histories of 1 commit each, git does not find a connection point between the 2 branches which makes them diverged branches from each other. But since in this case they are not in fact diverged I agree the explanation should say shallow clone and not diverged branches.